blueocean-web/src/main/js/main.jsx

```diff
@@ -37,17 +39,18 @@ class App extends Component {
     }
 
     render() {
+        const { t } = this.props;
         return (
             <div className="Site">
                 <header className="Site-header">
                     <div className="global-header">
                         <Extensions.Renderer extensionPoint="jenkins.logo.top"/>
                         <nav>
-                            <Link to="/pipelines">Pipelines</Link>
-                            <a href="#">Administration</a>
+                            <Link to="/pipelines">{t('pipelines')}</Link>
+                            <a href="#">{t('administration')}</a>
```

Comment:
I like the brevity of this syntax too, although I think we should just get a reference to "t" via import / require.